value.go

```diff
@@ -14,6 +14,87 @@ import (
 // Value represents all Javascript values and objects
 type Value struct {
 	ptr C.ValuePtr
+	ctx *Context
+}
+
+// NewValue will create a primitive value. Supported values types to create are:
+//   string -> V8::String
+//   int32 -> V8::Integer
+// 	 uint32 -> V8::Integer
+//	 bool -> V8::Boolean
+//   int64 -> V8::BigInt
+//   uint64 -> V8::BigInt
+//	 bool -> V8::Boolean
+// 	 *big.Int -> V8::BigInt
+func NewValue(iso *Isolate, val interface{}) (*Value, error) {
```

Comment:
this should probably include a nil check for iso

Reply:
Can do. Would fail pretty quickly without the VM running 😉 
Also noticed the poor formatting of the doc comment, will fix that too.